crates/astria-sequencer/src/app/test_utils.rs

```diff
@@ -101,8 +103,8 @@ pub(crate) async fn initialize_app_with_storage(
 
     let genesis_state = genesis_state.unwrap_or_else(|| GenesisState {
         accounts: default_genesis_accounts(),
-        authority_sudo_address: Address::from([0; 20]),
-        ibc_sudo_address: Address::from([0; 20]),
+        authority_sudo_address: address_from_hex_string(JUDY_ADDRESS),
+        ibc_sudo_address: address_from_hex_string(TED_ADDRESS),
```

Comment:
I added addresses in here that were different because changing the fee recipient was NOT causing insta tests to fail. The proposer & the authority address where both the nil addresses.

Reply:
yeah this is one of those things snapshots wouldn't necessarily catch since it's not modfying the logic of any action, this is a good add